### DISCUSSION TITLE
Osu Lazer crashes on startup (Mac M1)

### DISCUSSION THREAD

**Discussion by KisteVonKritz** [ppy/osu#16450] [Q&A] [answered]

**Comment by KisteVonKritz** [comment, marked as answer]
Ah, its working again, nvm...

**Comment by bdach** [comment]
do not open discussions like this if you are not going to do the bare minimum of [providing crash logs](https://github.com/ppy/osu/discussions/15188)

**Reply by KisteVonKritz** [reply]
I provided cash log, but when the game started working again, I deleted it to make sure, that somebody will close/delete this discussion. Sorry for inconvenience.
